data/mods/realguns/ammo.json

```diff
@@ -0,0 +1,238 @@
+[
+  {
+    "id": "40sw",
+    "copy-from": "40sw",
+    "type": "AMMO",
+    "name": ".40 S&W",
+    "name_plural": ".40 S&W JHP",
+    "description": ".40 S&W ammunition with 135gr JHP bullets.  The .40 S&W round is descended from the 10mm Auto light loading and maintains most of its predecessor's strengths while reducing recoil.",
+    "extend": { "ammo_type": [ "10mm_auto" ] }
+  },
+  {
+    "id": "40fmj",
+    "copy-from": "40fmj",
+    "type": "AMMO",
+    "name": ".40 S&W FMJ",
+    "name_plural": ".40 S&W FMJ",
+    "description": ".40 S&W ammunition with 180gr FMJ bullets.  Designed to retain the 10mm Auto cartridge's power with lower recoil, the .40 S&W round became popular for law enforcement and personal defense.",
+    "extend": { "ammo_type": [ "10mm_auto" ] }
+  },
+  {
+    "id": "reloaded_40sw",
+    "copy-from": "reloaded_40sw",
+    "type": "AMMO",
+    "name": "reloaded .40 S&W",
+    "name_plural": "reloaded .40 S&W JHP",
+    "description": ".40 S&W ammunition with 135gr JHP bullets.  The .40 S&W round is descended from the 10mm Auto light loading and maintains most of its predecessor's strengths while reducing recoil.",
+    "extend": { "ammo_type": [ "10mm_auto" ] }
+  },
+  {
+    "id": "reloaded_40fmj",
+    "copy-from": "reloaded_40fmj",
+    "type": "AMMO",
+    "name": "reloaded .40 S&W FMJ",
+    "description": ".40 S&W ammunition with 180gr FMJ bullets.  Designed to retain the 10mm Auto cartridge's power with lower recoil, the .40 S&W round became popular for law enforcement and personal defense.",
+    "extend": { "ammo_type": [ "10mm_auto" ] }
+  },
+  {
+    "id": "10mm",
+    "copy-from": "10mm",
+    "type": "AMMO",
+    "name": "10mm Auto",
+    "name_plural": "10mm Auto",
+    "delete": { "ammo_type": [ "40" ] }
+  },
+  {
+    "id": "reloaded_10mm",
+    "copy-from": "10mm",
+    "type": "AMMO",
+    "name": "reloaded 10mm Auto",
+    "name_plural": "reloaded 10mm Auto",
+    "proportional": {
+      "price": 0.7,
+      "damage": 0.9,
+      "dispersion": 1.1
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": {
+      "ammo_type": [ "40" ],
+      "effects": [ "NEVER_MISFIRES" ]
+    }
+  },
+  {
+    "id": "45lc_casing",
```

Comment:
See `data/json/items/generic/casing.json` for a better definition of cartridge casings